<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConvolve</class>
 <widget class="QDialog" name="DialogConvolve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convolve with Kernel</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="kernelTableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section { background-color:#D3D3D3 }</string>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="kernelSizeGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>106</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Kernel Size</string>
   </property>
   <widget class="QRadioButton" name="kernel3x3RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3×3 Kernel</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="kernel5x5RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5×5 Kernel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="multiplierLabel">
      <property name="text">
       <string>Multiplier:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="multiplierLineEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="borderComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Replicate</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reflect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reflect 101</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wrap</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="borderLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Border Policy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="presetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Presets:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Emboss</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Laplacian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian Blur 3×3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian Blur 5×5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PrewittX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PrewittY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sharpen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SobelX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SobelY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unsharp Masking 5×5</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConvolve</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConvolve</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
